src/Storage/ExerciseModels/Noun.fs

```diff
@@ -12,7 +14,7 @@ type Noun(id, model: Exercises.Noun) =
     new() = Noun(null, Exercises.Noun.Default)
 
     [<SerializeOption>] member val Gender = model.Gender with get, set
-    [<SerializeObject>] member val Patterns = model.Patterns with get, set
+    [<SerializeObject>] member val Patterns = model.Patterns |> Seq.map DeclensionPattern.toString with get, set
```

Comment:
This is kind of different from other fields on the storage models but there are not much we can do here (we cannot store DUs directly in the Azure Table Storage). 

I could go for reflection and implement another attribute but it would be oddly specific (doing whatever `toString` does for this type), defeating the purpose of attributes whatsoever.

I will soon do similar modifications for verb patterns and we can see how things look like.